sources/packages/backend/apps/workers/src/controllers/workflow-variables.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Created while starting the assessment-gateway workflow.
+ * Workflows: assessment-gateway.
+ * Type: Global variable.
```

Comment:
mm.. I didn't get what is `Type: Global variable.`

Reply:
Those comments were an attempt to try to point out where the variables are in the workflow. The global here means that it is used across the entire workflow, in this case, the `assessment-gateway`. Sometimes they can be created at the task-level only, so when the worker is listing the variables to be retrieved could be hard to figure out from where they are coming and I was trying to make it clear.
These comments are also a suggestion, if they do not help at all someone that will be doing changes in the workflows we can get rid of them 😉